MetaMask/metamask-mobile
# feat(2739): permission summary view gets the ability to disconnect all

## **Description**
The permission summary view gets the ability to diconnect all





## **Related issues**

Contributes to solve issue: https://github.com/MetaMask/MetaMask-planning/issues/2796

## **Manual testing steps**

I have added screenshot below to see how it looks.

1. The steps to get this modal to show are to have a dapp already connected and click the icon at the top right header
2. This will open the bottom sheet with the button to disconnect all.

## **Screenshots/Recordings**


| Before       | After      |
|--------------|--------------|
| <img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/4984cdeb-3ca2-4a7a-8df0-b90e5f2db987"> |<img width="350" alt="Screenshot 2024-04-18 at 3 56 43 PM" src="https://github.com/user-attachments/assets/51e47204-b6a0-4f7f-91f2-45c5d66b4a33"> |








## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
